.github/workflows/tests.yml

```diff
@@ -221,11 +221,12 @@ jobs:
         id: files
         uses: tj-actions/changed-files@v46
         with:
-          files: |
-            **/*.py
-            pyproject.toml
-          files_ignore: |
-            test/**
+          files_yaml: |
+            python:
+              - '**/*.py'
+              - '!test/**'
+            pyproject:
+              - 'pyproject.toml'
```

Comment:
This is unrelated: in #9199, I made the `lint` workflow trigger when pyproject is changed (to check type issues with mypy) but doing `pylint pyproject.toml` is not expected and causes errors.
Here I am fixing it.

Reply:
Done in #9217.